t/02_yaml_schedule.t

```diff
@@ -21,6 +21,17 @@ subtest 'parse_yaml_test_data_single_import' => sub {
 
 };
 
+subtest 'parse_yaml_test_data_vars_expansion' => sub {
+    use scheduler;
+    use testapi 'set_var';
+
+    set_var('ENV_VAR', 'my-env-details');
+    my $schedule = $ypp->load_file(dirname(__FILE__) . '/data/test_data_vars_expansion.yaml');
+    scheduler::parse_test_suite_data($schedule);
+    my $testdata = scheduler::get_test_suite_data();
+    ok $testdata->{name_with_env_details} eq 'name-my-env-details', "Test data value was expanded properly";
```

Comment:
I believe we should cover more cases than just one. At least some case with nested key, as well as array.

Reply:
done, good suggestion, thank you, I discovered some bug in my initial design.